src/content/docs/accounts/accounts-billing/account-setup/student-program.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: New Relic for Students 
+metaDescription: Sign up for a New Relic student account 
+freshnessValidatedDate: never
+---
+
+Are you a student who is trying to learn about gathering and analyzing hardware and software telemetry? You can do this with New Relic for Students. This is a program that offers you free access to data, users, and New Relic features&mdash;even more options than you'd get from the Free Edition. 
+
+## Benefits [#student-benefits]
+
+Eligible students will receive access to a free New Relic for Students account for two years, which includes:
+
+* 500GB of data ingest per month
+* 3 Full platform users
+* 10,000 synthetics checks a month
+* Access to all other features available to free product users
```

Comment:
Can we link to list of these features somewhere?

Reply:
@paperclypse I added links to most of these bullets, but some of generic because the anchors will change in a week.

I need to get confirmation about my change from "Free" to "Pro."

Reply:
I switched it to "Standard" per our SME.